docs/en/latest/plugins/azure-functions.md

```diff
@@ -104,20 +104,20 @@ Request-Context: appId=cid-v1:38aae829-293b-43c2-82c6-fa94aec0a071
 Date: Wed, 17 Nov 2021 14:46:55 GMT
 Server: APISIX/2.10.2
 
-Hello, apisix
+Hello, APISIX
 ```
 
-For requests where the mode of communication between the client and the Apache APISIX gateway is HTTP/2, the example looks like ( make sure you are running APISIX agent with `enable_http2: true` for a port in conf.yaml or uncomment port 9081 of `node_listen` field inside [config-default.yaml](../../../../conf/config-default.yaml) ) :
+For requests where the mode of communication between the client and the Apache APISIX gateway is HTTP/2, the example looks like ( make sure you are running APISIX agent with `enable_http2: true` for a port in [config-default.yaml](../../../../conf/config-default.yaml). You can do it by uncommenting the port 9081 from `apisix.node_listen` field ) :
```

Comment:
Please don't use a link to the `config-default.yaml`. Here is what the link looks like in the doc site:
http://apisix.apache.org/docs/apisix/plugins/azure-functions
http://apisix.apache.org/conf/config-default.yaml

Reply:
I see, I guess It's because of the relative path URL. Updated